integration-tests/manager-integration-tests/manager-integration-tests-v2/README.md

```diff
@@ -0,0 +1,141 @@
+On the other side sometimes in the test we need to reference these resources ids, i.e. to check its value in external systems.
+
+To combine both requirements we introduced a concept of test identifier and system identifier.
+Test identifier is a specific resource identifier used in a scenario (for example Bridge name). When a Bridge is created the test engine takes the test identifier, generates a unique system identifier corresponding to the test identifier and store the test to system identifier mapping in a context. System identifier is then used as a real value when interacting with remote system.
+
+### Feature file placeholders
+
+It is possible to use various placeholders in feature files, allowing dynamic access to runtime values and system identifiers. The placeholders use format `${placeholder-value}`.
+
+Currently, there is a possibility to use these placeholders in feature files:
+- ${aws.access-key} to use an AWS access key belonging to a particular AWS account to connect to
+- ${aws.region} to use a particular AWS region belonging to an AWS account to connect to
+- ${aws.secret-key} to use an AWS secret key belonging to a particular AWS account to connect to
+- ${aws.sqs.`<Queue name>`} to use an AWS SQS queue
+- ${bridge.`<Bridge name>`.endpoint.base} to use Bridge endpoint base URL (i.e. for http://localhost:80/something it will return http://localhost:80)
+- ${bridge.`<Bridge name>`.endpoint.path} to use Bridge endpoint path (i.e. for http://localhost:80/some/thing it will return /some/thing)
+- ${bridge.`<Bridge name>`.processor.`<Processor name>`.id} to use actual "Processor id" linked to a specific Bridge
+- ${bridge.`<Bridge name>`.id} to use actual "Bridge id"
+- ${cloud-event.`<Cloud event id>`.id} to use "System cloud event id" (Cloud event id which is actually used for Cloud event invocation)
+- ${data.`<Data id>`} to retrieve value stored in the context under this id
+- ${env.`<System property name>`} to use a System property
+- ${manager.authentication.token} to use a token for communication with Manager
+- ${slack.channel.`<Channel name>`.name} to use actual channel name
+- ${slack.channel.`<Channel name>`.webhook.url} to use a WebHook URL for sending message to the specified channel
+- ${slack.token} to use a token for communication with Slack
+- ${topic.`<Topic name>`} to use a unique Kafka topic name
+- ${uuid.`<Uuid name>`} to use unique identifier. Useful to distinguish historical data produced by the same test for example.
+- ${webhook.site.token.`<Token name>`} to use a webhook.site token with specified alias
+
+## Test execution
+
+### Local testing
+
+If you deployed application locally (by [running manager and operator locally](../../dev/README.md)) you can use [run-local-tests.sh](run-local-tests.sh) script to launch the tests with all local parameters set automatically.
```

Comment:
same as above

Reply:
Similar change is required in `readme` for v1 as well

Reply:
Thanks, updated.